ui/js/component/fileTile/view.jsx

```diff
@@ -126,17 +126,22 @@ class FileTile extends React.PureComponent {
                   {" "}
                   {showLocal && fileInfo && <Icon icon={icons.LOCAL} />}
                 </span>
-                <div className="meta">{uri}</div>
                 <h3>
                   <TruncatedText lines={1}>{title}</TruncatedText>
                 </h3>
               </div>
               {description &&
                 <div className="card__content card__subtext">
-                  <TruncatedText lines={!showActions ? 4 : 2}>
+                  <TruncatedText lines={2}>
                     {description}
                   </TruncatedText>
                 </div>}
+              <div className="card__content card__subtext">
```

Comment:
I agree we ought to make a change here to give a stronger indication how to open these.

But what if the change should just be to match card behavior of discover page? These card elements are really quite similar, so we should just be consistent in the effect/UI we provide.

Reply:
> add experimental fileTile styles 87c4bd2 
Ok I was just testing / experimenting with some stuff but yeah I'll remove it

Reply:
Agree, but a tile and a card are two different things...

Reply:
>  we should just be consistent in the effect/UI

Reply:
Would you considering removing the `uri` from the tile component ?

Reply:
we could add a menu / or a set of actions to the tile component...

Reply:
a small button `Get uri` or `Share`